src/test/java/com/google/cloud/tools/libraries/LibrariesTest.java

```diff
@@ -43,20 +62,66 @@ public void testJson() throws FileNotFoundException, URISyntaxException {
     JsonReader reader = factory.createReader(in); 
     Iterator<JsonValue> apis = reader.readArray().iterator();
     Assert.assertTrue(apis.hasNext());
-    for (JsonObject api = (JsonObject) apis.next(); apis.hasNext(); api = (JsonObject) apis.next()) {
+    while (apis.hasNext()) { 
+      JsonObject api = (JsonObject) apis.next();
+      verifyApi(api);
+    }
+  }
+
+  private static final String[] statuses = {"alpha", "beta", "GA"};
+
+  private static void verifyApi(JsonObject api) throws URISyntaxException {
```

Comment:
nit : would using assertApi instead of verifyApi clarify that the intention of the method is to make assertions?